src/assets/inline-svgs/logos/facebook-logo.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="28" height="28" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M27.9993 13.7918C27.9993 6.174 21.7321 0 13.9993 0C6.26646 0 -0.000732422 6.174 -0.000732422 13.7918C-0.000732422 20.677 5.11802 26.3823 11.8118 27.4167V17.7785H8.25708V13.7918H11.8118V10.7533C11.8118 7.29728 13.9008 5.38744 17.1 5.38744C18.6313 5.38744 20.2336 5.65681 20.2336 5.65681V9.05089H18.4672C16.7282 9.05089 16.1868 10.1149 16.1868 11.2059V13.7918H20.0696L19.4489 17.7785H16.1868V27.4167C22.8805 26.3823 27.9993 20.677 27.9993 13.7918Z" fill="#277056"/>
+<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 28 28">
+  <path d="M28 13.792C28 6.174 21.731 0 14 0 6.265 0 0 6.174 0 13.792c0 6.885 5.118 12.59 11.812 13.625v-9.639H8.257v-3.986h3.555v-3.039c0-3.456 2.089-5.366 5.288-5.366 1.531 0 3.134.27 3.134.27V9.05h-1.767c-1.739 0-2.28 1.064-2.28 2.155v2.586h3.883l-.621 3.987h-3.262v9.638c6.694-1.035 11.812-6.74 11.812-13.625Z" fill="currentColor"/>
```

Comment:
These social logo SVG changes are
- Run the SVG through [svgOmg](https://jakearchibald.github.io/svgomg/)
- Set the path fills to "currentColor" so we can color them using tailwind classes